mmaction/datasets/pipelines/loading.py

```diff
@@ -910,6 +910,122 @@ def __call__(self, results):
 
 
 @PIPELINES.register_module()
+class AudioDecodeInit(object):
+    """Using librosa to initialize the audio reader.
+
+    Args:
+        io_backend (str): io backend where frames are store.
+            Default: 'disk'.
+        sample_rate (int): Audio sampling times per second. Default: 16000.
+
+    Required keys are "audio_path", added or modified keys are "length",
+    "sample_rate", "audios".
+    """
+
+    def __init__(self, io_backend='disk', sample_rate=16000, **kwargs):
+        self.io_backend = io_backend
+        self.sample_rate = sample_rate
+
+        self.kwargs = kwargs
+        self.file_client = None
+
+    def __call__(self, results):
+        """Perform the librosa initialization.
+
+        Args:
+            results (dict): The resulting dict to be modified and passed
+                to the next transform in pipeline.
+        """
+        try:
+            import librosa
+        except ImportError:
+            raise ImportError('Please install librosa first.')
+
+        if self.file_client is None:
+            self.file_client = FileClient(self.io_backend, **self.kwargs)
+        try:
+            file_obj = io.BytesIO(self.file_client.get(results['audio_path']))
+            y, sr = librosa.load(file_obj, sr=self.sample_rate)
+        except BaseException:
+            # Generate a ramdom dummy 10s input
```

Comment:
is this used for debugging?

Reply:
Nah, around 1/20 videos in Kinetics400 do not have audio stream. This is a systematic issue rather than an arbitrarily occuring error.